spec/computable/registry/claim-reward.spec.ts

```diff
@@ -0,0 +1,129 @@
+import * as ganache from 'ganache-cli'
+import Web3 from 'web3'
+import { Contract, Block } from 'web3/types.d'
+import { stringToBytes, increaseTime } from '../../helpers'
+import Erc20 from '../../../src/contracts/erc-20'
+import Voting from '../../../src/contracts/plcr-voting'
+import Parameterizer from '../../../src/contracts/parameterizer'
+import Registry from '../../../src/contracts/registry'
+import { ParameterDefaults, NAME } from '../../../src/constants'
+import {
+  deployDll,
+  deployAttributeStore,
+  maybeParseInt,
+  eventReturnValues,
+} from '../../../src/helpers'
+
+const provider:any = ganache.provider(),
```

Comment:
this is the old http-provider style, now deprecated. you'll need to setup the newer style with a beforeAll and AfterAll to get the websocket-provider for web3.

Reply:
Ok, think I got this figured out.